app/views/users/new.html.haml

```diff
@@ -0,0 +1,24 @@
+%h1 Sign Up
+%br
+= form_for @user do |f|
+  = f.label :first_name
+  %br
+  = f.text_field :first_name
+  %br
+  = f.label :last_name
+  %br
+  = f.text_field :last_name
+  %br
+  = f.label :email
+  %br
+  = f.text_field :email
+  %br
+  = f.label :phone_number
+  %br
+  = f.telephone_field :phone_number
+  %br
+  = f.label :password
```

Comment:
@njgheorghita Let's add the password_confirmation, unless we have a strong reason not to?  Otherwise it seems like a user could potentially sign up, but fat-finger their password, and then not be able to log in again...